### ISSUE TITLE
find method notebook data grid wrapping issue

### ISSUE THREAD

**Issue by zieka** [moderneinc/moderne-visualizations-misc#1]
when the find method notebook is rendered as a data grid in the ui there is a weird wrapping thing

**Comment by sjungling** [comment]
Example:

Running _Find Method_ on `moderneinc/moderne-common` for `java.util.List add(..)` will result in 2 rows for this source code:

Source:

```
  edges.add(new DefaultEdge<>(nodes.get(i),
          new DefaultConnectionCursor(Integer.toString(limitOffset.getOffset() + i))));
```

Data table row produced by recipe: (ignore smile encoding)
```
���sourceFile�src/main/java/io/moderne/common/graphql/ExtendedRelayConnection.java��method�edges.add(new DefaultEdge<>(nodes.get(i),
        new DefaultConnectionCursor(Integer.toString(limitOffset.getOffset() + i))))���@�src/main/java/io/moderne/common/monitoring/MonitoringAutoConfiguration.java�Aotags.add(Tag.of("application", applicationName))��
```

Seems like it is something that would need to be addressed in the underlying [recipe](https://github.com/openrewrite/rewrite/blob/main/rewrite-java/src/main/java/org/openrewrite/java/search/FindMethods.java#L79) with how it produces data table rows.

**Comment by zieka** [comment]
related to https://github.com/moderneinc/code-data-science/commit/049ca931df9502fb90f39e75264b2ec92ffd78ad

**Comment by zieka** [comment]
fixed in version `0.1.27`
